Assignment2/VictorNS69-Victor Nieves.rdf

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<rdf:RDF 
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+    <rdf:Description rdf:about="#User10A">
+		<hasName>Pedro</hasName>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Class01">
+		<includes rdf:resource="#Sensor029"/>
+		<includes rdf:resource="#Computer101"/>
+    </rdf:Description>	
+    <rdf:Description rdf:about="#Computer101">
+		<hasOwner rdf:resource="#User10A"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+		<hasMeasurement rdf:resource="#Measurement8401"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Measurement8401">
+		<hasTemperature>29</hasTemperature>
+		<atTime>2010-06-12T12:00:12</atTime>
```

Comment:
It is good practice to datatype these literals.

Reply:
Is this now ok?

https://github.com/VictorNS69/Curso2019-2020/blob/e54b85b3442dd1f857e415610e8d2206f6fff9e1/Assignment2/VictorNS69-Victor%20Nieves.rdf#L16